templates/rapidsai-core/partials/install_notebooks.dockerfile.j2

```diff
@@ -6,14 +6,6 @@ RUN gpuci_conda_retry install -y -n rapids \
     && gpuci_conda_retry remove -y -n rapids --force-remove \
         "rapids-notebook-env=${RAPIDS_VER}*"
 
-{# Install jupyter lab stuff #}
-RUN gpuci_conda_retry install -y -n rapids jupyterlab-nvdashboard
-
-RUN source activate rapids \
-  && jupyter labextension install @jupyter-widgets/jupyterlab-manager dask-labextension jupyterlab-nvdashboard \
```

Comment:
I think we still need `@jupyter-widgets/jupyterlab-manager` and `dask-labextension` to be installed this way. It's just nvdashboard which can be installed via conda for now.

Reply:
@jacobtomlinson, from the two doc pages below, I was under the impression that these extensions only need to be installed via `pip` or `conda` in JupyterLab 3. So I added them as dependencies in https://github.com/rapidsai/integration/pull/266. Can you take a look and tell me if you agree?

- https://github.com/dask/dask-labextension#jupyterlab-30-or-greater
- https://www.npmjs.com/package/@jupyter-widgets/jupyterlab-manager

Reply:
Ah exciting! Then yes you're right.

Are we adding those pip installed somewhere else?

Reply:
> Are we adding those pip installed somewhere else?

Do we need anything `pip` installed? I thought the `conda` installation of those packages was enough.

Reply:
Sure, I meant `pip/conda` installed.

Reply:
Gotcha. The `dask-labextension` package seems to be already included in our `notebook-env` [here](https://github.com/rapidsai/integration/blob/branch-0.20/conda/recipes/rapids-notebook-env/meta.yaml#L44). I added the `jupyterlab_widgets` package in https://github.com/rapidsai/integration/pull/266/, which seems to be the equivalent of [@jupyter-widgets/jupyterlab-manager](https://www.npmjs.com/package/@jupyter-widgets/jupyterlab-manager). But actually, after discovering and reading the [docs](https://ipywidgets.readthedocs.io/en/stable/user_install.html#installing-in-jupyterlab-3-0) for `@jupyter-widgets/jupyterlab-manager`, it seems that maybe I should add the `ipywidgets` package to that PR as well.

Reply:
Added `ipywidgets` in https://github.com/rapidsai/integration/pull/266/commits/3c581f6534c11a4a6ec3fd55a8a4ad0d0d8eb7d6. I think that covers everything. Let me know if you agree

Reply:
That looks great thanks!